library/psa_crypto_slot_management.h

```diff
@@ -203,12 +203,16 @@ psa_status_t psa_validate_key_location( psa_key_lifetime_t lifetime,
 /** Validate the persistence of a key.
  *
  * \param[in] lifetime  The key lifetime attribute.
+ * \param[in] key_id    The key identifier.
  *
  * \retval #PSA_SUCCESS
+ * \retval #PSA_ERROR_INVALID_HANDLE
+ *         \p key_id is not a valid key identifier.
```

Comment:
There is `psa_validate_key_id()` to validate key identifiers thus why do you introduce such validation in this function that validates the persistence of a key ?